cmd/minishift/cmd/delete.go

```diff
@@ -46,23 +48,41 @@ func runDelete(cmd *cobra.Command, args []string) {
 
 	util.ExitIfUndefined(api, constants.MachineName)
 
+	if err := cluster.UnRegister(api); err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error unregistering the VM: %s", err.Error()))
+	}
+
+	if minishiftConfig.InstanceConfig.IsRegister {
```

Comment:
@hferentschik if unregister already happen then this doesn't even run because `unregister` will set this flag to false. We are using this flag for a situation only (below) for any other situation we don't actually warn user every time about `vm is registered`

1. If a VM is stopped but not unregistered (using --skip-unregister)
2. If a VM crashed and user going to delete it.